@xen-orchestra/web-core/lib/components/link/VtsLink.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <component :is="component" :class="classes" class="vts-link" v-bind="attributes">
+    <UiIcon :icon="icon" color="current" />
+    <slot />
+    <UiIcon
+      v-if="attributes.target === '_blank'"
+      :icon="faArrowUpRightFromSquare"
+      class="external-icon"
+      color="current"
+    />
+  </component>
+</template>
+
+<script lang="ts" setup>
+import UiIcon from '@core/components/icon/UiIcon.vue'
+import { useLinkComponent } from '@core/composables/link-component.composable'
+import { toVariants } from '@core/utils/to-variants.util'
+import type { IconDefinition } from '@fortawesome/fontawesome-common-types'
+import { faArrowUpRightFromSquare } from '@fortawesome/free-solid-svg-icons'
+import { computed } from 'vue'
+import type { RouteLocationRaw } from 'vue-router'
+
+const props = defineProps<{
+  size: 'small' | 'medium'
+  accent: 'brand' | 'success' | 'warning' | 'danger'
+  icon?: IconDefinition
+  disabled?: boolean
+  href?: string
+  to?: RouteLocationRaw
+  target?: '_blank' | '_self'
+}>()
+
+const typoClasses = {
+  small: 'typo p3-regular',
+  medium: 'typo p1-regular',
```

Comment:
In the typography styles, there is a `-underline` variant, what do you think of using it? This would avoid adding the `text-decoration` CSS property and match the typography used in the design system:
```suggestion
  small: 'typo p3-regular-underline',
  medium: 'typo p1-regular-underline',
```